pkg/apis/streaming/v1alpha1/processor_types.go

```diff
@@ -75,6 +75,23 @@ type StreamBinding struct {
 	Alias string `json:"alias,omitempty"`
 }
 
+const (
+	Earliest = "earliest"
+	Latest   = "latest"
+)
+
+type InputStreamBinding struct {
+	// Stream name, from this namespace, to be bound to the processor
+	Stream string `json:"stream"`
+
+	// Alias exposes the stream under another name within the processor
+	// +optional
+	Alias string `json:"alias,omitempty"`
+
+	// Where to start consuming this stream the first time a processor runs.
+	StartOffset string `json:"startOffset"`
```

Comment:
missing `// +optional`

Reply:
It works with that fix

Reply:
Should work without it as there is a defaulter. Moreover, this is only a spec and code snippet was here to give a "feeling" of the change. Better to discuss this in the `system` PR if at all

Reply:
We are in the system PR ^^
Re-tested this morning without the `// +optional` and it works now.